pkg/core/native/oracle.go

```diff
@@ -534,8 +534,8 @@ func (o *Oracle) getConvertibleFromDAO(d *dao.Simple, key []byte, item stackitem
 
 // updateCache updates cached Oracle values if they've been changed.
 func (o *Oracle) updateCache(d *dao.Simple) error {
-	orc, _ := o.Module.Load().(OracleService)
-	if orc == nil {
+	orc, _ := o.Module.Load().(*OracleService)
+	if orc == nil || *orc == nil {
```

Comment:
Can we have a race condition, where all oracle services are restarted simultaneously and some requests are skipped?

Reply:
Probably. The only way to solve it is to fetch currently active requests on service start, I'll take a look at it.

Reply:
It's the same problem for all currently active requests before node sync now, they'll be completely ignored.

Reply:
Fixed.